workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/I18n/Language.json

```diff
@@ -197,6 +198,7 @@
 	"find-a-project": "Find a project",
 	"finish-activation": "Finish Activation",
 	"first-name": "First Name",
+	"fix-version": "Fix Version",
 	"fix-versions": "Fix Versions",
```

Comment:
if going singular for most frontend labeling, can we clean up any of the plural naming we arent using anymore?

Reply:
we still use the plural keys on the individual ticket page. If we want to change/remove them we can, but i think it makes sense in that context still